docs/try-it-out.mdx

```diff
@@ -15,16 +15,16 @@ doc_type: getting-started
 displayed_sidebar: mainSidebar
 ---
 
-Explore what's possible using the Beyond Identity platform before having to write any code.
+See what you can do with Beyond Identity before you write any code.
 
-The three steps below allow you to: 
```

Comment:
The number of steps may change. Removing the number reduces the risk of future errors.